src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets

```diff
@@ -1,7 +1,9 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   
   <PropertyGroup>
-    <TodayTimeStamp>$([System.DateTime]::Now.ToString(yyyyMMdd))</TodayTimeStamp>
+    <!-- If OfficialBuildId is passed in, then use it as Today's TimeStamp, else default to today -->
+    <TodayTimeStamp Condition="'$(OfficialBuildId)'!=''">$(OfficialBuildId)</TodayTimeStamp>
```

Comment:
Currently TodayTimeStamp is an 8 digit # like 20160715.  Official Build IDs are like 20160715-01.  Going in like this will break the build pipeline (easily fixed though)